### PR TITLE
DHFPROD-6261: Page Size functionality for the Load List pagination is broken.

### PR DESCRIPTION
**Pull request by timur-isangulov**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/load/load-list.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { convertDateFromISO } from '../../util/conversionFunctions';
 import Steps from "../steps/steps";
 import {AdvLoadTooltips, SecurityTooltips} from "../../config/tooltips.config";
 import { MLTooltip } from '@marklogic/design-system';
+import { LoadingContext } from '../../util/loading-context';
```
Comment (wooldridge) [inline marklogic-data-hub-central/ui/src/components/load/load-list.tsx:14]: @timur-isangulov, I'm seeing an error that loading-context can't be found and I don't see it here:

https://github.com/timur-isangulov/marklogic-data-hub/tree/DHFPROD-6261/marklogic-data-hub-central/ui/src/util

(There might also be an issue with the file path here.)

**Review by wooldridge — CHANGES_REQUESTED** [review]

**Review by wooldridge — APPROVED** [review]

File: marklogic-data-hub-central/ui/src/pages/Load.tsx [hunk 2]
```diff
@@ -38,8 +36,6 @@ const Load: React.FC = () => {
   };
 
   useEffect(() => {
-    setPage(location.state?.page);
```
Comment (briantang) [inline marklogic-data-hub-central/ui/src/pages/Load.tsx:41]: Pagination looks good but I see you undid some of my recent work that was needed for preserving pagination when cancelling 'Add to New Flow'. Was it a necessary change for this ticket?

https://drive.google.com/file/d/1VIKQ0OfuiTSqONtrZfP_Ra28Kpyxc3ZN/view

**Review by briantang — CHANGES_REQUESTED** [review]

**Review by wooldridge — COMMENTED** [review]
Looks good, unit tests pass, manual testing with 21 load steps works fine: 

https://drive.google.com/file/d/1jRJT8pkKCl5PMcTm2McKs90iNxXam1O7/view

File: marklogic-data-hub-central/ui/src/pages/Load.tsx (hunk 2 above)
Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/pages/Load.tsx:41]: @briantang Can you please review again.

**Review by briantang — APPROVED** [review]
Looks good now!

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**